connector_ecommerce/sale.py

```diff
@@ -291,12 +291,10 @@ def action_view_parent(self, cr, uid, ids, context=None):
         action['res_id'] = parent.id
         return action
 
-    # TODO: remove in odoo 8.0
+    # XXX the 3 next methods seems very specific to magento
```

Comment:
Not really, but I have an obsolete copy and I overwrite the code. I haven't done the migration, and don't have a look in depth of the code, so I'm going also to act here as a reviewer/fixer :wink: